libraries/AP_DroneCAN/AP_DroneCAN.h

```diff
@@ -223,6 +223,9 @@ class AP_DroneCAN : public AP_CANDriver, public AP_ESC_Telem_Backend {
     uint32_t _SRV_last_send_us;
     HAL_Semaphore SRV_sem;
 
+    AP_HAL::Util::safety_state saftey_state_last;
```

Comment:
These should probably be initialised somewhere?

Reply:
Sure, why not

Reply:
the variable is not needed at all if we do it based on time since the last change

Reply:
removed